### PR TITLE
Publish InfluxDB Enterprise 1.9.3 docs

### LINKED ISSUE #2831: Publish InfluxDB Enterprise 1.9.3 documentation

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2831.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -9,6 +9,27 @@ menu:
...
+### Bugfixes
+- Improve heap memory usage when HH queue grows.
+- Avoid rewriting `fields.idx` unnecessarily.
+- Do not close connection twice in `DigestWithOptions`.
+- Do not panic on cleaning up failed iterators.
+- Rename ARM RPMs with `yum`-compatible names.
+- Convert ARM arch names for RPMs during builds via Docker.
+- Do not send non-UTF-8 characters to subscriptions.
+- Error instead of panic for statement rewrite failure.
+- `show shards` gives empty expiry time for inf duration shards.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:31]: Ah! Need to fix this.

**Review by timhallinflux — APPROVED** [review]

File: content/enterprise_influxdb/v1.9/administration/configure-password-hashing.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    weight: 80
+    parent: Administration
+related:
+    - /enterprise_influxdb/v1.9/administration/configuration/
+---
+
+
+By default, InfluxDB Enterprise uses `bcrypt` for password hashing.
+[FIPS] compliance requires particular hashing alorithms.
+Use `pbkdf2-sha256` or `pbkdf2-sha256` for FIPS compliance.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/administration/configure-password-hashing.md:17]:
```suggestion
Use `pbkdf2-sha256` or `pbkdf2-sha512` for FIPS compliance.
```

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 3]
```diff
@@ -9,6 +9,27 @@ menu:
...
 
+## v1.9.3 [2021-07-19]
+
+### Features
+- Add [configurable password hashing](/enterprise_influxdb/v1.9/administration/configure-password-hashing/) with `bcrypt` and `pbkdf2` support.
+- Add retry with exponential back-off to anti-entropy repair.
+- Add logging to compaction.
+- Add [`total-buffer-bytes`](/enterprise_influxdb/v1.9/administration/config-data-nodes/#total-buffer-bytes--0) configuration parameter to subscriptions.
+  This option is intended to help alleviate out-of-memory errors.
+- Update Flux to v0.120.1.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:20]:
```suggestion
- Update to [Flux v0.120.1.](/influxdb/v2.0/reference/release-notes/flux/#v01201-2021-07-06)
```

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 4]
```diff
@@ -9,6 +9,27 @@ menu:
...
+
+### Bugfixes
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:22]:
```suggestion
### Bug fixes
```

**Review by kelseiv — APPROVED** [review]

**Pull request merged**